docs/fips.md

```diff
@@ -0,0 +1,80 @@
+# FIPS compliance
+
+<!----Should we mention the version when we implemented FIPS? ----->
+
+The Federal Information Processing Standards (FIPS) are a set of U.S. government standards that ensure the security of computer systems for non-military government agencies and contractors. These standards specify how to perform cryptographic operations, such as encryption, hashing, and digital signatures. FIPS mode is a mode of operation that enforces these standards and rejects any non-compliant algorithms or parameters.
+
+Percona Server for MySQL implements the same level of FIPS support as MySQL. Percona Server for MySQL can run in FIPS mode if a FIPS-enabled OpenSSL library and FIPS Object Module are available at runtime or if compiled using a FIPS-validated version of OpenSSL. You can also receive this functionality by [building Percona Server for MySQL from source code](compile-percona-server.md).
+
+## Prerequisites
+
+To prepare Percona Server for MySQL for FIPS certification, do the following:
+
+* Check that your operating system includes FIPS pre-approved OpenSSL library in version 3.0.x or higher. The following distributions includes FIPS pre-approved OpenSSL library in version 3.0.x or higher:
+
+    * RedHat Enterprise Linux 9
+
+    * Oracle Linux 9
```

Comment:
>    * RedHat Enterprise Linux 9

>    * Oracle Linux 9

@percona-ysorokin maybe it is better to write RedHat Enterprise Linux 9 and derivatives here?

Reply:
I have tested on OL9